tests/integration/test_ec2.py

```diff
@@ -13,23 +13,52 @@
 from awscli.testutils import unittest, aws
 
 
-class BaseEC2Test(unittest.TestCase):
-    def assert_dry_run_success(self, command):
+class TestDescribeInstances(unittest.TestCase):
+    def setUp(self):
+        self.prefix = 'ec2 describe-instances --region us-west-2'
+
+    def test_describe_instances_with_id(self):
+        command = self.prefix + ' --instance-ids malformed-id'
         result = aws(command)
-        expected_response = ('Request would have succeeded, '
-                             'but DryRun flag is set.')
-        self.assertIn(expected_response, result.stderr)
+        self.assertIn('InvalidInstanceID.Malformed', result.stderr)
```

Comment:
Question: If you use MaxResults and provide a malformed instance id, what error gets propogated?  I have the same question for the rest of the test cases. Might be good to add a note or rename the tests as without any background knowledge it is not obvious why we are testing for that specific error, other than for the sake of a quick smoke test of the service.

Especially since for the case, where you have no instances matching the filter, EC2 will actually not complain about MaxResults being present:
```
$ aws ec2 describe-instances --filters Name=instance-id,Values=malformed --max-items 5
{
    "Reservations": []
}

```
That being said I am not sure if there is a great way to test the filter case.